client/app/dedicated/server/bandwidth/bandwidth.service.js

```diff
@@ -0,0 +1,75 @@
+class BaseDedicatedService {
```

Comment:
Could maybe be extracted to its own file ?

Reply:
Related to my other comment, it's an existing service that was moved. I'm not refactoring it with this feature, since it's used on others pages :(